modules/administration-guide/pages/configuring-a-user-namespace.adoc

```diff
@@ -0,0 +1,96 @@
+:_content-type: PROCEDURE
+:description: Configuring a user namespace
+:keywords: administration guide, configuring, user, namespace
+:navtitle: Configuring a user namespace
+:page-aliases:
+
+[id="configuring-a-user-namespace"]
+= Configuring a user namespace
+
+This procedure walks through the process of using the {prod-short}
+to replicate `ConfigMaps`, `Secrets` and `PersistentVolumeClaim` from `{prod-namespace}` namespace to numerous
+user-specific namespaces. The operator automates the synchronization of important configuration
+data such as passwords, SSH keys, and certificates to user namespaces.
+
+If a {kubernetes} resource is changed in an {prod-namespace} namespace,
+the operator will immediately replicate the changes across all users namespaces.
+In reverse, if a {kubernetes} resource is modified in a user namespace,
+the operator will immediately revert the changes.
+
+The special logic is used for 'PersistentVolumeClaim' resources. If a 'PersistentVolumeClaim'
+is updated in {prod-namespace} or user namespaces, it will not be replicated or reverted due
```

Comment:
```suggestion
Special logic is used for 'PersistentVolumeClaim' resources. If you update 'PersistentVolumeClaim'
in {prod-namespace} or user namespaces, it will not be replicated or reverted due
```